### PR TITLE
Add method NumericHelper::isInteger

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: tests/NumericHelperTest.php [hunk 1]
```diff
@@ -66,4 +66,28 @@ public function testNormalizeNumberWithIncorrectType(): void
         $this->expectException(\InvalidArgumentException::class);
         NumericHelper::normalize([]);
     }
+
+    public function dataIsInteger(): array
+    {
+        return [
+            ['42', true],
+            ['-42', true],
+            ['0', true],
```
Comment (roxblnfk) [inline tests/NumericHelperTest.php:75]: What about case with value `'042'`? It will be convert to `'42'` and `isInteger` returns false. Is it correct?

Comment (vjik) [reply tests/NumericHelperTest.php:75]: Fixed

File: src/NumericHelper.php [hunk 2]
```diff
@@ -71,6 +71,6 @@ public static function normalize($value): string
      */
     public static function isInteger(string $value): bool
```
Comment (roxblnfk) [inline src/NumericHelper.php:72]: Do you think the second parameter to detect unsigned int is needed here?

Comment (vjik) [reply src/NumericHelper.php:72]: I don't have use cases.

Comment (roxblnfk) [reply src/NumericHelper.php:72]: @yiisoft/yii3 what do you think?

Comment (xepozz) [reply src/NumericHelper.php:72]: Better to add separated method when is will be needed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 ## 1.1.1 under development
 
 - Enh #62: Add method `StringHelper::split` that split a string to array with non-empty lines (vjik)
+- Enh #63: Add method `NumericHelper::isInteger` that checks whether the given string is integer number (vjik)
```
Comment (samdark) [inline CHANGELOG.md:7]:
```suggestion
- Enh #63: Add method `NumericHelper::isInteger` that checks whether the given string is an integer number (vjik)
```

File: src/NumericHelper.php [hunk 4]
```diff
@@ -61,4 +61,16 @@ public static function normalize($value): string
         $value = str_replace([' ', ','], ['', '.'], (string)$value);
         return preg_replace('/\.(?=.*\.)/', '', $value);
     }
+
+    /**
+     * Checks whether the given string is integer number
+     *
+     * @param string $value
+     *
+     * @return bool
```
Comment (samdark) [inline src/NumericHelper.php:70]:
```suggestion
     * Checks whether the given string is an integer number.
```

File: src/NumericHelper.php [hunk 5]
```diff
@@ -61,4 +61,16 @@ public static function normalize($value): string
...
+     */
+    public static function isInteger(string $value): bool
```
Comment (samdark) [inline src/NumericHelper.php:72]:
```suggestion
    public static function isInteger($value): bool
```

It is convenient to be able to pass real integers here.

File: src/NumericHelper.php [hunk 6]
```diff
@@ -61,4 +61,16 @@ public static function normalize($value): string
...
+    {
+        return preg_match('/^-?\d+$/', $value) === 1;
```
Comment (samdark) [inline src/NumericHelper.php:74]:
```suggestion
        return filter_var($value, FILTER_VALIDATE_INT); 
```

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**